src/github.com/matrix-org/dendrite/clientapi/routing/getevent.go

```diff
@@ -0,0 +1,151 @@
+// Copyright 2017 Vector Creations Ltd
```

Comment:
It's 2019! Also please put yourself here instead of Vector Creations Ltd, as you've written the code :)

Reply:
Ah, they've been changed. Thanks!